src/App.jsx

```diff
@@ -1,33 +1,14 @@
 import React from 'react';
-import { BrowserRouter, Route, Routes } from 'react-router-dom';
-import Navbar from './components/Navbar';
-import Home from './views/Home';
-import ArtistList from './views/ArtistList';
-import ArtistDetail from './views/ArtistDetail';
-import NotFound from './views/NotFound';
-import Register from './views/Register';
-import Login from './views/Login';
-import AuthContextProvider from './contexts/AuthContext';
+import { BrowserRouter } from 'react-router-dom';
+import AppRoutes from './AppRoutes';
 
 const App = function () {
   return (
-
     <BrowserRouter>
-      <AuthContextProvider>
-        <Navbar />
-        <main>
-          <Routes>
-            <Route index element={<Home />} />
-            <Route path="register" element={<Register />} />
-            <Route path="login" element={<Login />} />
-            <Route path="artists" element={<ArtistList />} />
-            <Route path="artists/:id" element={<ArtistDetail />} />
-            <Route path="*" element={<NotFound />} />
-          </Routes>
-        </main>
-      </AuthContextProvider>
+      <main>
+        <AppRoutes />
+      </main>
```

Comment:
Este cambio está alterando la estructura que teníamos previamente. No sé si afectará en el estilo, pero lo que sí ocurre es que `<main</main>` encierra todo ahora, incluso el header (en `<NavBar />`) y no debería, pues el header es independiente de main.

Deberíamos mantener la estructura original, es decir algo como lo siguiente:
```suggestion
      <AppRoutes />
```
Y que `<AppRoutes />` incluya todo lo original, desde `<AuthContextProvider>` como componente que encierra todo, pasando con `Navbar` y `main` como hijos, y luego `<Routes>` dentro de `main`.